Resources/Prototypes/_RMC14/Entities/Clothing/Belt/belts.yml

```
@@ -335,3 +335,21 @@
   - type: Tag
     tags:
     - RMCSmartPistolHolster
+
+- type: entity
+  parent: CMBeltBaseStorage
+  id: RMCBeltUtilityGeneral
+  name: M276 G8-A general utility pouch
+  definition: A small, lightweight pouch that can be clipped onto M3 Pattern armor to provide additional storage. The newer G8-A model, while uncomfortable, can also be clipped around the waist.
+  components:
+  - type: Sprite
+    sprite: _RMC14/Objects/Clothing/Belt/sparepouch.rsi
+  - type: Clothing
+    slots:
+    - suitStorage
+    - belt
+    sprite: _RMC14/Objects/Clothing/Belt/sparepouch.rsi
+  - type: Storage
+    maxItemSize: Normal
+    grid:
+    - 0,0,8,1 # 9 slots
```

0,0,8,1 is 4.5 slots, not 9 (each slot is 2 wide and it starts counting from 0), but that aside this belt isn't supposed to use slots.
From 13 it's supposed to have 10 storage space, so 0,0,9,1
```suggestion
  - type: Storage
    maxItemSize: Normal
    grid:
    - 0,0,9,1 # 9 slots
```

---

Ah, was going by the wiki and that said it can store 3 items, so i assumed it meant 3 normal sized items, which is 9 tiny items; the comment there was to reflect that, if a bit inaccurately.
Applied the changes now